src/main/typescript/node_modules/@atomist/rug/tree/TreeHelper.ts

```diff
@@ -66,3 +66,15 @@ export class TreeHelper {
         return r
     }
 }
+
+export abstract class AddressedNodeSupport implements Addressed {
```

Comment:
Maybe it's all the Scala, but semicolons just add noise IMO. Also, the vast majority of our generated and library TS code doesn't use them. Either we change everywhere or not.